plate_simulation/models/params.py

```diff
@@ -101,7 +102,7 @@ def center(
         :param surface: surface object to reference plate depth from.
         :param depth_offset: Additional offset to be added to the depth of the plate.
         """
-        return self._get_xy(survey) + [self._get_z(surface, depth_offset)]
+        return [*self._get_xy(survey), self._get_z(surface, depth_offset)]
```

Comment:
Interesting syntax - never seen this before.

Reply:
unpacking array